src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -420,6 +441,8 @@ public unsafe void Open()
                     _glfw.WindowHint(WindowHintClientApi.ClientApi, ClientApi.OpenGLES);
                     break;
             }
+            
+            _glfw.WindowHint(WindowHintBool.Visible, _initialOptions.IsVisible);

```

Comment:
Can you double check that all proprerties are used within this method in some way? Visibility might not be the only property we didn't send to GLFW...